dataactbroker/handlers/fileHandler.py

```diff
@@ -290,7 +303,8 @@ def getStatus(self):
                     else:
                         jobInfo["duplicated_headers"] = []
                     jobInfo["error_type"] = self.interfaces.errorDb.getErrorType(jobId)
-                    jobInfo["error_data"] = self.interfaces.errorDb.getErrorMetricsByJobId(jobId,self.jobManager.getJobType(jobId)=='validation',self.interfaces)
+                    jobInfo["error_data"] = self.interfaces.errorDb.getErrorMetricsByJobId(jobId,self.jobManager.getJobType(jobId)=='validation',self.interfaces, severityId=self.interfaces.validationDb.getRuleSeverityId("fatal"))
```

Comment:
Modified this function to only get job type once per job, as above the severity ID dict is cached after the first call.